Sources/StreamVideoSwiftUI/CallView/VideoRenderer.swift

```diff
@@ -188,11 +214,11 @@ extension VideoRenderer {
             )
             add(track: track)
             DispatchQueue.global(qos: .userInteractive).asyncAfter(deadline: .now() + 0.01) { [weak self] in
-                guard let self else { return }
+                guard let self = self else { return }
                 let prev = participant.trackSize
-                if let viewSize, prev != viewSize {
+                if let viewSize = self.viewSize, prev != viewSize {
```

Comment:
Is this needed for iOS 14 issues?

Reply:
I think it was one of the things i tried. I'll remove to keep the diff only to meaninful changes